app/src/main/java/org/openobservatory/ooniprobe/test/TestAsyncTask.java

```diff
@@ -151,6 +151,10 @@ private void runTest(AbstractTest... tests) {
                     currentTest.run(app, app.getPreferenceManager(),app.getLogger(), app.getGson(), result, i, this);
                 }
             }
+            if (result.countTotalMeasurements() == tests.length) {
```

Comment:
Where is this function defined?

Maybe this function should be called "countTotalCompletedMeasurements" or something of that sort.

Also, what exactly is counted here? Is it really a measurements or is it a nettest in a group?

For example if you run `web_connectivity` against 10 URLs and then `whatsaapp`, would this count be 11 or would it be 2?

Reply:
11